bool x_gripperOpened
bool x_gripperClosed
